docs/products/postgresql/reference/idle-connections.rst

```diff
@@ -0,0 +1,26 @@
+Idle connections
+================
+
+How Aiven for PostgreSQL® handles idle database connection sessions and uses TCP keep-alive.
```

Comment:
Can we state more about what the article covers? and also give a link to idle database connections maybe to PG docs?

Reply:
I have made a change and added a link for the server-side keepalive parameters.